test/test.js

```diff
@@ -962,4 +962,18 @@ asyncTest('Loading a System.registerdynamic module (not bundled)', function() {
   }).then(null, err);
 });
 
+asyncTest('Importing a script with wrong integrity fails', function() {
```

Comment:
Because it's just checking if it doesn't load, this should load fine in Node (an error is thrown when it can't verify the "integrity" prop.